src/univariate/discrete/skellam.jl

```diff
@@ -79,9 +81,14 @@ function cf(d::Skellam, t::Real)
     μ1, μ2 = params(d)
     exp(μ1 * (cis(t) - 1) + μ2 * (cis(-t) - 1))
 end
+#based on SciPy: https://github.com/scipy/scipy/blob/v0.15.1/scipy/stats/_discrete_distns.py
+function cdf(d::Skellam, t::Real)
+    μ1, μ2 = params(d)
+    t = floor(t)
+    (t < 0) ? nchisqcdf(2*μ2, -2*t, 2*μ1) : 1.0 - nchisqcdf(2*μ1, 2*(t+1), 2*μ2)
+end
 
-cdf(d::Skellam, x::Int) = throw(MethodError(cdf, (d, x)))
-cdf(d::Skellam, x::Real) = throw(MethodError(cdf, (d, x)))
+cdf(d::Skellam, t::Int) = cdf(d, float(t))
```

Comment:
```julia
cdf(d::Skellam, t::Int) = cdf(d, float(t))
```

Is not required, the signature above will also work

Reply:
It was throwing some error, that's why I added it. Will look once more then.

Reply:
There are a few tests in `test/ref/discrete_test.ref.json`

Reply:
Should I add more?

Reply:
hadn't seen it sorry, no it's fine :)

Reply:
Okay!